visualization/index.html

```diff
@@ -135,39 +135,39 @@ <h1>
 			<section id="legend" class="container">
 				<div id="dv_infobox" class="row">
 					<div class="column_3">
-						<h2>Cómo leer la visualización</h2>
+						<h2>How to read the display</h2>
```

Comment:
I'm not sure display is the best translation... visualization? chart?

Reply:
Fixed it with "visualization", since it's used in the very next line.